### PR TITLE
WIP React Table v7 update

### PR DESCRIPTION
**Pull request by srkirkland**
Updating to the new hooks based React Tables v7.  Created wrapper around hooks table component called `ReactTable` which takes in columns + data.  All existing tables are broken except the People table has been fixed up to work with the new code (but needs styling help).  Most tables will be easy to convert with the exception of custom header filters (like the expiration dropdown, for which you should take a look at https://github.com/tannerlinsley/react-table/blob/master/docs/examples/simple.md for an example, and also in our existing Filtering.tsx file).

### CODE REVIEW

File: Keas.Mvc/ClientApp/components/People/PeopleTable.tsx [hunk 1]
```diff
@@ -68,7 +86,7 @@ export default class PeopleTable extends React.Component<IProps, {}> {
 
     const initialState: Partial<TableState<any>> = {
       sortBy: [{ id: 'name' }],
-      pageSize: ReactTableUtil.getPageSize(),
+      pageSize: ReactTableUtil.getPageSize()
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/People/PeopleTable.tsx:89]: Didn't we decide to go with trailing commas?

Comment (srkirkland) [reply Keas.Mvc/ClientApp/components/People/PeopleTable.tsx:71]: If we did, we didn't codify it in the peaks config file

**Comment by srkirkland** [comment]
Added filtering, so the remaining tables should be pretty easy to fix up with the exception of the custom column filters (like for serials or expiration date). This example shows how to do it https://codesandbox.io/embed/github/tannerlinsley/react-table/tree/master/examples/filtering

**Comment by srkirkland** [comment]
closed in favor of #899

**Pull request closed without merging**